staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -147,3 +154,34 @@ type CustomResourceValidation struct {
 	// OpenAPIV3Schema is the OpenAPI v3 schema to be validated against.
 	OpenAPIV3Schema *JSONSchemaProps `json:"openAPIV3Schema,omitempty" protobuf:"bytes,1,opt,name=openAPIV3Schema"`
 }
+
+// CustomResourceSubResources defines the status and scale subresources for CustomResources.
+type CustomResourceSubResources struct {
+	// Status denotes the status subresource for CustomResources
```

Comment:
include documentation about what this actually does when set. For example:
```
* exposes a /status subresource for the custom resource
* PUT requests to the /status subresource take a custom resource object, and ignore changes to anything except the status stanza
* PUT/POST/PATCH requests to the custom resource ignore changes to the status stanza
```

Reply:
Done.